.github/workflows/rust.yml

```
@@ -38,8 +40,41 @@ jobs:
+      - name: Run tests for core library
+        run: cargo +${{ env.MSRV }} test
+      - name: Run tests for std feature
+        run: cargo +${{ env.MSRV }} test --features std
+      - name: Run tests for extensions
+        run: cargo +${{ env.MSRV }} test --features extensions --lib extensions -- --test-threads=1
+      - name: Run doc tests
+        run: cargo +${{ env.MSRV }} test --doc --all-features
+
   test:
-    needs: lint
+    needs: [ lint, msrv ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)